recipes/bilby_tgr/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "bilby_tgr" %}
+{% set version = "0.1a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bilby_tgr-{{ version }}.tar.gz
+  sha256: 7074eeff19da652449a2e2539d932506ffad3dd8ccde5cd9c5eebb1fe15b4613
+  patches:
+    - 0001-minimise-dependencies.patch
```

Comment:
Can you add a comment here to why that patch is needed? For our future self sake ;-p

Reply:
Thank you @ocefpaf ! I added a comment on the patch, according to the discussion above. (Sorry for the slow reply, everyone was on vacation, I think.)